__tests__/password-reset-handler.test.ts

```
@@ -0,0 +1,129 @@
+    
+    expect(supabaseClient.auth.resetPasswordForEmail).toHaveBeenCalledWith(
+      'test@example.com',
+      { redirectTo: 'https://testero.ai/reset-password' }
+    );
+    
+    process.env.NEXT_PUBLIC_SITE_URL = originalEnv;
+  });
+
+  it('falls back to localhost when NEXT_PUBLIC_SITE_URL is not set', async () => {
+    const originalEnv = process.env.NEXT_PUBLIC_SITE_URL;
+    delete process.env.NEXT_PUBLIC_SITE_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)